irc/commands.go

```diff
@@ -90,6 +90,10 @@ func init() {
 			usablePreReg: true,
 			minParams:    1,
 		},
+		"CHATHISTORY": {
```

Comment:
Hmm. Can we change this to `draft/CHATHISTORY` for now (and the associated help entry)? Especially given the spec's gone through some decently-radical overhauls lately, pays to be a bit cautious with the command name itself for the time being methinks.

Reply:
Oh, actually... No let's not do this here.